README.md

```diff
@@ -231,51 +217,33 @@ change this line in `mix.exs`:
 defp elixirc_paths(:test), do: ["lib", "web", "test/support", "test/factories"]
 ```
 
-## Using without Ecto
+## Custom strategies (using ExMachina without Ecto)
 
-You can use ExMachina without Ecto, by using just the `build` function, or by
-defining `save_record/1` in your module.
+You can use ExMachina without Ecto, by using just the `build` functions, or you
+can define one or more custom strategies to use in your factory. Here's an
+example, but check out the docs on ExMachina.Strategy for more info.
 
 ```elixir
-defmodule MyApp.JsonFactory do
-  use ExMachina
-
-  def factory(:user) do
-    %User{name: "John"}
-  end
+defmodule MyApp.JsonEncodeStrategy do
+  use ExMachina.Strategy, function_name: :json_encode
 
-  def save_record(record) do
-    # Poison is a library for working with JSON
+  def handle_json_encode(record, _opts) do
     Poison.encode!(record)
   end
 end
 
-# Will build and then return a JSON encoded version of the map
-MyApp.JsonFactories.create(:user)
-```
-
-You can do something similar while also using Ecto by defining a new function.
-This gives you the power to call `create` and save to Ecto, or call `build_json`
-or `create_json` to return encoded JSON objects.
-
-```elixir
-defmodule MyApp.Factory do
-  use ExMachina.Ecto, repo: MyApp.Repo
+defmodule MyApp.JsonFactory do
```

Comment:
Can we just call this `MyApp.Factory`? I think the `JsonFactory` calls itself out a little too much. Took me a couple seconds to realize it wasn't actually doing any magic and it's irrelevant.